tests/test_models/test_utils.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import numpy as np
+
+from floodlight.models.utils import _exclude_x_ids
+
+
+@pytest.mark.unit
+def test_exclude_x_ids(example_xy_object_geometry_convex_hull) -> None:
+    # Arrange
+    xy = example_xy_object_geometry_convex_hull
+    xIDs_to_exclude = [0, 4]
+
+    # Act
+    include_x_ids = _exclude_x_ids(xy, xIDs_to_exclude)
+
+    # Assert
+    assert np.array_equal(
+        include_x_ids,
+        np.array([False, False, True, True, True, True, True, True, False, False]),
+    )
```

Comment:
If this test was more rigorous, i.e., different scenarios (list of xIDs empty, of wrong data type, index out of bounds, etc.), we wouldn't really need different unit tests for the fit method parameterized with different values of ```exclude_xIDs```, or am I missing something?